.github/workflows/instascale-release.yml

```diff
@@ -13,10 +13,14 @@ on:
         description: 'Published version of multi-cluster-app-dispatcher to be used as a dependency for InstaScale'
         required: true
         default: 'v0.0.0-dev'
-      is-latest:
-        description: 'Select if the built image should be tagged as latest'
+      is-stable:
+        description: 'Select if the built image should be tagged as stable'
         required: true
         type: boolean
+      branch:
+        description: 'Branch to be used for tag and release'
+        required: true
+        default: 'main'
```

Comment:
https://github.com/actions/checkout#usage

This is unnecessary. The `checkout` action defaults to the ref that the GitHub action is running for